backend/src/main/java/ai/giskard/service/SliceService.java

```diff
@@ -91,8 +91,11 @@ private static void writeSliceResultToCacheFile(Path cachedSliceFile, List<Integ
     }
 
     private void validateSliceName(String sliceName) {
-        if (sliceName.length() > 30) {
-            throw new IllegalArgumentException(String.format("Slice name %s is too long. Slice names should not be longer than 30 characters.", sliceName));
+        if (sliceName.length() < 1) {
```

Comment:
We should check if the sliceName is not blank to avoid potential validation of name like `  `:

```suggestion
        if (sliceName == null || sliceName.trim().isEmpty()) {
```